src/point_one/fusion_engine/common/portability.h

```diff
@@ -40,11 +40,21 @@
 typedef int32_t p1_ssize_t;
 #elif defined(_MSC_VER)
 typedef int64_t p1_ssize_t;
+// Definitions for armcc compiler.
+#elif defined(__MST__)
+#  include "typedefs.h"
+typedef int32_t p1_ssize_t;
 #else
```

Comment:
I'd hope the arm.compiler actually had an arch depending ssizet...crazy that it doesn't.